apps/api/src/app/activity/usecases/get-acticity-graph-states/get-acticity-graph-states.command.ts

```diff
@@ -1,12 +1,7 @@
 import { IsNumber, IsOptional } from 'class-validator';
-import { CommandHelper } from '../../../shared/commands/command.helper';
 import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
 
 export class GetActivityGraphStatsCommand extends EnvironmentWithUserCommand {
-  static create(data: GetActivityGraphStatsCommand) {
-    return CommandHelper.create<GetActivityGraphStatsCommand>(GetActivityGraphStatsCommand, data);
-  }
```

Comment:
from all the commands I've removed this part of code that is duplicated across

Reply:
This is amazing, so excited about this!!